### ISSUE TITLE
No import

### ISSUE THREAD

**Issue by Sentinel-7** [Kookabura/msYmlImport#1]
Please write how to use it?
Need to import from yml file to minishop2

**Comment by Kookabura** [comment]
Вам нужна папка cron. Внутри нее файл importer.php. Его запускаете из командной строки. Перед запуском нужно в таблице с товаром сделать поле externalKey.
Это очень примерно. Подробной инструкции нет.

**Comment by Sentinel-7** [comment]
а в importer.php этот foreach куда кладет данные? в опции minishop2?
foreach ($offer->param as $param) {
            switch ($val = (string)$param['name']) {
                case "Длина":
                    $data['length'] = (float)$param;
                    break;

**Comment by Kookabura** [comment]
Да, эти поля тоже должны быть в таблице, т.е ее расширять нужно

**Comment by Sentinel-7** [comment]
Все отработало хорошо, но картинки не загрузились, подскажите в чем ошибка?
http://picplus.ru/img/1903/20/c71d2ce5.png

**Comment by Kookabura** [comment]
Не знаю, скорее всего амазон как-то блокирует

**Comment by Sentinel-7** [comment]
Да нет.. из браузера открывает все ок.. 
http://hubber-test.s3-website-us-east-1.amazonaws.com/b12b1976-7eb4-470b-9a05-4e46a5fe37c1.jpg

**Comment by Sentinel-7** [comment]
А что такое externalKey?
И у меня несколько фото для товара, может из-за этого ошибка?
http://picplus.ru/img/1903/20/f6ef07d2.png

**Comment by Kookabura** [comment]
Это id товара в yml. Нужно для обновления. Мы его в таблицу site_content добавляли. И инициализировали с помощью плагина. Без этого обновить товары не получится.

Несколько картинок не должно быть проблемой.

**Comment by Sentinel-7** [comment]
А плагин который в компоненте здесь?

**Comment by Kookabura** [comment]
Не нашел, похоже, что его нет. Помню, что он был коротким. Что внутри точно не помню, надо искать.

**Comment by Sentinel-7** [comment]
эх... :(((

**Comment by Sentinel-7** [comment]
К сожалению да, не обновляет товары.. т.е. повторный импорт добавляет такие же новые... 
Если Вам не трудно, помогите с решением пожалуйста

**Comment by Kookabura** [comment]
Это несложно, но отнимет какое-то время. Ход решения я рассказал, вам осталось код написать.
Можете сами поискать решение или договориться об условиях моего участия.
Моя почта: aseregin@tolkit.top
